flask/test/auth_tests/test_auth.py

```diff
@@ -0,0 +1,50 @@
+from auth_tests.auth import get_user_token_string
+import pytest
+from unittest.mock import patch
+
+get_auth_string_patch = patch("boxwise_flask.auth_helper.get_auth_string_from_header", get_user_token_string)
+get_auth_string_patch.start()
```

Comment:
Why is it that patching happens here and in `patches.py`? Can we only do either?

Reply:
this patch is only required by the auth tests. I've updated it to clean it up a little